### DISCUSSION TITLE
ColossalAI-Examples中webtextdataset

### DISCUSSION THREAD

**Discussion by weiwei0519** [hpcaitech/ColossalAI#2838] [Community | Q&A]
ColossalAI-Examples中from datasets.webtext import WebtextDataset，这个数据集在哪个package，我尝试安装了openai的dataset，和huggingface的datasets，都没有找到

**Comment by feifeibear** [comment]
https://github.com/hpcaitech/ColossalAI/blob/main/examples/language/gpt/titans/dataset/webtext.py
请看ColossalAI的example/gpt/titans的如上文件
